qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/BackupErrorResponseTest.java

```diff
@@ -139,7 +139,7 @@ final class RequestTimeoutTest {
     private BackupEndpoint backupEndpoint;
 
     private void lowerTimeout(final GatewayCfg config) {
-      config.getCluster().setRequestTimeout(Duration.ofMillis(1));
+      config.getCluster().setRequestTimeout(Duration.ofNanos(500));
```

Comment:
:thought_balloon: I'm not sure if this is the actual cause. `400` indicates broker is receiving the request and rejecting with an `unsupported_message` error because backup store is configured. In this test, we are disconnecting broker 0 from the gateway, which means broker should never receive the message. So a lower timeout should not matter. The root cause would be to understand why broker 0 is receiving the message, even though we disconnect it. May be disconnect is not working as expected. Can you look into it?